cupyx/scipy/signal/_spectral.py

```diff
@@ -1,6 +1,27 @@
+"""
+Spectral analysis functions and utilities.
+
+Some of the functions defined here were ported directly from CuSignal under
+terms of the Apache license, under the following notice
```

Comment:
@andfoy this info looks incorrect and seems important to fix. cuSignal was explicitly relicensed to MIT in order to facilitate upstreaming into CuPy. See https://github.com/rapidsai/cusignal/pull/587#issuecomment-1690346502 and gh-7816.

Could you please open a follow-up PR to address this?